melodic/distribution.yaml

```diff
@@ -8544,15 +8544,13 @@ repositories:
       version: melodic-devel
     release:
       packages:
-      - pilz_extensions
       - pilz_industrial_motion
       - pilz_robot_programming
       - pilz_store_positions
-      - pilz_trajectory_generation
```

Comment:
Just so you understand the implications of removing these packages; once we merge this, both `pilz_extensions` and `pilz_trajectory_generation` will be removed completely from the repositories, and won't be able to be installed anymore.  Are you OK with that?

Reply:
@clalancette Thank you for pointing this out. This was not intended.